info.yml

```diff
@@ -73,23 +73,43 @@ configuration:
                      if they are defined."
 
     slate_logo:
-        type: config_path
-        description: This is the path to an image to use on the slate such as a
+        type: template
+        fields: context
+        description: This is the template for a path to an image to use on the slate such as a
```

Comment:
Make it a template so that it gets a show location by default and can be overridgen in sgtk_config_templates if needed (this will be a problem if the user switches seq/shot within the application as opposed to doing `go`. Not sure if we can work around that.)

Reply:
I updated tk-multi-publish2 and tk-multi-workfiles2 to handle this condition, and we should probably add similar functionality to the new file/open file scene hooks, but in general since this is a show location, I doubt the value will change as artists should not be working across shows.